tests/callbacks_test.cpp

```diff
@@ -470,24 +495,25 @@ TEST_CASE("SACallbacksFullFunctionTest", "[CallbacksTest]")
 /**
  * Make sure the EarlyStopAtMinLoss callback will stop the optimization process.
  */
-TEST_CASE("EarlyStopAtMinLossCallbackTest", "[CallbacksTest]")
-{
-  SGDTestFunction f;
-  arma::mat coordinates = f.GetInitialPoint();
-
-  // Instantiate the optimizer with a number of iterations that will take a
-  // long time to finish.
-  StandardSGD s(0.0003, 1, 2000000000, -10);
-  s.ExactObjective() = true;
-
-  // The optimization process should return in one second.
-  const double result = s.Optimize(f, coordinates, EarlyStopAtMinLoss(100));
-
-  REQUIRE(result == Approx(-1.0).epsilon(0.0005));
-  REQUIRE(coordinates(0) == Approx(0.0).margin(1e-3));
-  REQUIRE(coordinates(1) == Approx(0.0).margin(1e-7));
-  REQUIRE(coordinates(2) == Approx(0.0).margin(1e-7));
-}
+//TEST_CASE("EarlyStopAtMinLossCallbackTest", "[CallbacksTest]")
+//{
+  
+  // SGDTestFunction f;
+  // arma::mat coordinates = f.GetInitialPoint();
+
+  // // Instantiate the optimizer with a number of iterations that will take a
+  // // long time to finish.
+  // StandardSGD s(0.0003, 1, 2000000000, -10);
+  // s.ExactObjective() = true;
+
+  // // The optimization process should return in one second.
+  // const double result = s.Optimize(f, coordinates, EarlyStopAtMinLoss(100));
+
+  // REQUIRE(result == Approx(-1.0).epsilon(0.0005));
+  // REQUIRE(coordinates(0) == Approx(0.0).margin(1e-3));
+  // REQUIRE(coordinates(1) == Approx(0.0).margin(1e-7));
+  // REQUIRE(coordinates(2) == Approx(0.0).margin(1e-7));
```

Comment:
I have commented this one, if it is no longer required I will remove it

Reply:
I think that we need to keep this test---I don't see a reason to remove it.  Maybe I overlooked something?